Assets/Scripts/EditorState/UnityState.cs

```diff
@@ -1,7 +1,7 @@
+using DynamicPanels;
```

Comment:
This changed happend because saving the document reordered the imports. The `using DynamicPanels` import is necessary for the field `dynamicPanelsCanvas` in line 27.